AUTHORS.md

```diff
@@ -0,0 +1,7 @@
+# Authors
```

Comment:
Can you please also enrich `README.md` so that it resembles the usual style in our other repositories such as `reana-server`?